misc/website/README

```diff
@@ -0,0 +1,38 @@
+The script build_website.py builds the website locally, combining the following components:
+    - docs/ directory of the Fast Downward repository (= this repository)
+    - documentation of the search plugins generated by
+      misc/autodoc/generate-docs.py
+    - documentation from a clone of https://github.com/aibasel/downward-markdown
+
+The content is generated from the current local working copies of the repositories.
+In contrast to the public website, the generated website only contains the
+pages for this version (not for previous releases).
+
+Setup:
+------
+
+Create a python virtual environment and activate it:
+$ python -m venv .env
+$ source .env/bin/activate
+
+Install the requirements:
+$ python3 -m pip install -r requirements.txt
+
+Set the environment variable DOWNWARD_MARKDOWN to a local copy of the
+downward-markdown repository (https://github.com/aibasel/downward-markdown).
+
+$ export DOWNWARD_MARKDOWN=<path to local working copy>
+
+
+Usage:
+------
+Build the website with the following command:
+$ ./build_website.py
+
+It creates a directory "build". To see the website, enter this directory and
+run mkdocs (you can of course use a different port):
```

Comment:
Same here, no newline would be more consistent.

Reply:
Same here (change made).